src/atom-environment.js

```diff
@@ -1430,6 +1443,33 @@ or use Pane::saveItemAs for programmatic saving.`)
       await Promise.all(fileOpenPromises)
     }
 
+    if (missingFolders.length > 0) {
+      let message = 'Unable to open project folder'
+      if (missingFolders.length > 1) {
+        message += 's'
+      }
+
+      let description = 'The '
+      if (missingFolders.length === 1) {
+        description += 'directory `'
+        description += missingFolders[0].pathToOpen
+        description += '` does not exist.'
+      } else if (missingFolders.length === 2) {
```

Comment:
Not a big deal, but you could probably just include this case into the `else` by trimming the `, ` off of the end of the description string when `length === 2` before adding the "and" part, but tomato tomahto :)